### PR TITLE
feat: follow action and friend list

### PR DESCRIPTION
**Pull request by y-ykcir**
#### What this PR does / why we need it:

- [x] feat: follow action and friend list
- [x] refactor router

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: service/follow_service/follow_action.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	"fmt"
+)
+
+const (
+	FOLLOW = iota + 1
+	CANCEL
+)
+
+var (
+	ErrIvdAct    = errors.New("undefined action")
```
Comment (Lan-ce-lot) [inline service/follow_service/follow_action.go:15]: Maybe we can use a file to store all the custom errors separately.

**Review by Lan-ce-lot — APPROVED** [review]
LGTM

**Comment by Lan-ce-lot** [comment]
Please solve the conflict and lint.🫡

File: router/router.go [hunk 2]
```diff
@@ -20,49 +20,36 @@ func Init() *gin.Engine {
...
+		{
+			messageGroup.POST("/action/", message.SendMessageHandler)
+			messageGroup.GET("/chat/", message.QueryMessageHandler)
 		}
-		c.JSON(http.StatusOK, userInfoResp)
-	})
+		// comment路由组
+		commentGroup := douyinGroup.Group("/comment")
+		{
+			commentGroup.POST("/action/", comment.Action)
```
Comment (Lan-ce-lot) [inline router/router.go:49]: Some action need `middleware.JWTMiddleWare()`, such as comment action.

**Review by Lan-ce-lot — CHANGES_REQUESTED** [review]

File: service/follow_service/follow_action.go (hunk 1 above)
Comment (y-ykcir) [reply service/follow_service/follow_action.go:15]: Maybe do this in another pr

**Pull request merged**